open-metadata-implementation/adapters/authentication-plugins/http-helper/src/main/java/org/odpi/openmetadata/http/HttpHelper.java

```diff
@@ -1,52 +0,0 @@
-/* SPDX-License-Identifier: Apache-2.0 */
```

Comment:
This needs to stay as it handles the skip client checks on JVM level as suppose to be. At least for UI.